life_expectancy/tests/test_file_handlers.py

```diff
@@ -0,0 +1,41 @@
+"""
+Tests for the file handlers module.
+"""
+
+import os
+from unittest import mock
+
+import pandas as pd
+
+from life_expectancy.file_handlers import load_data, save_data
+
+from . import FIXTURES_DIR
+
+
+def test_load_data():
+    """
+    Run unit test of function `load_data`.
+    """
+    actual_data = load_data(FIXTURES_DIR / "dummy_dataset.tsv")
+    expected_data = pd.DataFrame.from_dict(
+        {
+            "col_1,col_2,col_3": ["1,5,23", "1,5,39", "x1,x2,x3"],
+            "2021": [79, 61, 70],
+            "2020": [85, 60, 50],
+        }
+    )
+
+    pd.testing.assert_frame_equal(actual_data, expected_data)
+
+
+def test_save_data():
+    """
+    Run unit test of function `save_data`.
+    """
+    test_df = pd.DataFrame.from_dict({"col_1": [1, 2, 3], "col_2": [1, 5, 6]})
+    with mock.patch("pandas.DataFrame.to_csv") as to_csv_mock:
```

Comment:
```suggestion
    with mock.patch("life_expectancy.file_handlers.pd.to_csv") as to_csv_mock:
```
It's usually better to try to be specific when defining what to patch. By patching "pandas.DataFrame" you are affecting all the dataframe instances in your code. My suggestion has a smaller blast radius: it only affects the dataframes from the file_handlers module